src/MongoDB.Driver.Core/Core/NativeLibraryLoader/RelativeLibraryLocatorBase.cs

```diff
@@ -53,6 +53,7 @@ private string GetAbsolutePath(string relativePath)
             var absolutePathsToCheck = new[]
             {
                 Path.Combine(libraryBasePath, libraryName),  // look in the current assembly folder
+                Path.Combine(libraryBasePath, @"..\..\", relativePath),
```

Comment:
NOTE: I'm not sure whether this logic(adding a new relative path) should be added here or for example on the `Zstandard64NativeMethods.cs` level (where we may return an array with relative paths). But since this updated relative path will be required for all native libraries, I've placed it on this level